app/src/App/Controller/Administrator/Activate.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Controller\Administrator;
+
+use Demo\Application\Service\Administrator\ActivateAdministrator;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+
+class Activate
+{
+    public function __construct(
+        private ActivateAdministrator $setActive
+    ) {
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        $administratorId = (int)$request->get('id');
```

Comment:
we use an empty space between casting and value by convention
```suggestion
        $administratorId = (int) $request->get('id');
```